providers/notificationProvider.tsx

```diff
@@ -0,0 +1,62 @@
+import * as Toast from '@radix-ui/react-toast';
+import {
+  createContext,
+  useEffect,
+  useState,
+  type CSSProperties,
```

Comment:
Oh what happened here? Why all these type imports?

And ESLint will complain about this as type imports should be separated from regular imports 🙈

Reply:
updated